ex7_building_modeling_optimization.ipynb

```diff
@@ -65,7 +65,7 @@
     "import os\n",
     "from scipy import optimize\n",
     "import numpy as np\n",
-    "from PyGEL3D import gel, js"
+    "from pygel3d import hmesh, jupyter_display"
```

Comment:
This fixes PyGEL3D on Linux, nice!

The display has the wrong up axis, but at least we can see the model ;-)
![image](https://user-images.githubusercontent.com/1630632/116362018-e546fb00-a801-11eb-967f-1aba9e0fbe0f.png)